kafka-connect-jms/src/test/scala/com/datamountaineer/streamreactor/connect/sink/writer/JMSWriterTest.scala

```diff
@@ -117,6 +123,10 @@ class JMSWriterTest extends TestBase with Using with BeforeAndAfter with Convert
 
```

Comment:
I guess that Thread.sleep has triggered you a little :D ...

Reply:
yes, it did ;)